series_solution.ipynb

```diff
@@ -0,0 +1,262 @@
+{
```

Comment:
<p>your return value is 0 and not the value in question</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>